cmd/src/users_prune.go

```diff
@@ -19,21 +19,21 @@ This command removes users from a Sourcegraph instance who have been inactive fo
 	
 Examples:
 
-	$ src users clean -days 182
+	$ src users prune -days 182
```

Comment:
How are such command/API changes usually made in `src` cli? Should this be a major version increase or should we keep backward compatibility leaving both `clean` and `prune` commands and marking one deprecated?

Reply:
Almost all the functionality changes in terms of the command run locally, the old `clean` command had some big timeout issues see https://github.com/sourcegraph/src-cli/issues/848. The name change from `clean` to `prune` was mostly just a style change. I don't think this command is being widely used so I think the change is alright. To my knowledge no one has put this on a cronjob or anything like that yet.